aws/lambda/blueprint.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: xl/v1
+kind: Blueprint
+metadata:
+  projectName: AWS Lambda and Credentials
+  description: |
+    This blueprint will configure one AWS Lamba application and one AWS credential set.
+  author: XebiaScrubs
+  version: 1.0
+parameters:
+- name: AwsInfrastructureDirectory
+  type: Input
+  description: "Infrastructure directory (default: AmazonAWS)"
+  default: "AmazonAWS"
+- name: AwsInfrastructureName
+  type: Input
+  description: "Infrastructure credential pair name (default: AmzAccount)"
+  default: AmzAccount
+- name: AwsEnvironmentDirectory
+  type: Input
+  description: 'Environment directory (default: aws-staging)'
+  default: aws-staging
+- name: AwsEnvironmentName
+  type: Input
+  description: 'Environment name (default: "AWSHosts")'
+  default: AWSHosts
+- name: AwsAccessKeyID
```

Comment:
get these automatically, check the other AWS blueprints to see how its done https://github.com/xebialabs/blueprints/blob/development/aws/monolith/blueprint.yaml